README.md

```diff
@@ -12,13 +12,32 @@ The [Atomic Developer Bundle](https://github.com/projectatomic/adb-atomic-develo
 
 ## Quick Start
 
-1. Install `vagrant-service-manager` plugin
+1. Install `vagrant-service-manager` plugin:
 
         vagrant plugin install vagrant-service-manager
 
-2. Get the [Vagrantfile](Vagrantfile) and start the ADB using `vagrant up`. [More](https://github.com/projectatomic/adb-atomic-developer-bundle/blob/master/docs/installing.rst) documentation on setting up ADB.
+2. Get the [Vagrantfile](Vagrantfile).
 
-3. Run the plugin to get environment variables and certificates
+3. Enable your desired provider(s) in [Vagrantfile](Vagrantfile) as:
+  ```
+    Vagrant.configure(2) do |config|
+
+      config.vm.box = "projectatomic/adb"
+
+      config.vm.network "private_network", type: "dhcp"
```

Comment:
This example should probably be the only line cited.

The Vagrantfile should contain comments with the list and format.